src/types/login.ts

```diff
@@ -1,28 +1,23 @@
 import { Document, Model } from 'mongoose'
 
+import { FormBillingStatistic, LoginBase } from '../../shared/types/billing'
+
 import { IAgencySchema } from './agency'
-import { AuthType, IFormSchema, IPopulatedForm } from './form'
+import { IFormSchema, IPopulatedForm } from './form'
 import { IUserSchema } from './user'
 
-export interface ILogin {
+export interface ILogin extends LoginBase {
   admin: IUserSchema['_id']
   form: IFormSchema['_id']
   agency: IAgencySchema['_id']
-  authType: AuthType
-  esrvcId: string
-  created?: Date
 }
 
-export interface ILoginSchema extends ILogin, Document {}
-
-export type LoginStatistic = {
-  adminEmail: IUserSchema['email']
-  formName: IFormSchema['title']
-  total: number
-  formId: IFormSchema['_id']
-  authType: ILoginSchema['authType']
+export interface ILoginSchema extends ILogin, Document {
+  created?: Date
```

Comment:
By moving `created` to `ILoginSchema`, is it still optional?

Reply:
Yes, since this typing is used for document creation, and we don't want to enforce a `created` prop to be assigned for document creation.

We should actually create a `LoginDocument` typing for the object that is retrieved from the database (with all the defaults populated, and have the optional types converted to required), but there are currently no cases where that is needed and thus not done yet.